lib/srv/desktop/windows_server.go

```diff
@@ -858,6 +861,17 @@ func (s *WindowsService) connectRDP(ctx context.Context, log *slog.Logger, tdpCo
 	tdpConn.OnSend = s.makeTDPSendHandler(ctx, recorder, delay, tdpConn, audit)
 	tdpConn.OnRecv = s.makeTDPReceiveHandler(ctx, recorder, delay, tdpConn, audit)
 	width, height := desktop.GetScreenSize()
+
+	computerName, ok := desktop.GetLabel(types.DiscoveryLabelWindowsDNSHostName)
+	if !ok {
+		computerName = strings.Split(desktop.GetAddr(), ":")[0]
+	}
+
+	kdcAddr := s.cfg.KDCAddr
+	if kdcAddr == "" {
+		kdcAddr = strings.Split(s.cfg.LDAPConfig.Addr, ":")[0]
+	}
```

Comment:
Same here, use a function built for parsing network addresses rather than a generic string function.

Also consider using `kdcAddr := cmp.Or(kdcAddr, ...)`.

Reply:
I went with `utils.Host` and changed the condition, `cmp.Or` won't work now